man/tpm2_sign.1.md

```diff
@@ -2,20 +2,29 @@
 
 # NAME
 
-**tpm2_sign**(1) - Sign a hash using the TPM.
+**tpm2_sign**(1) - Sign a hash or message using the TPM.
 
 # SYNOPSIS
 
 **tpm2_sign** [*OPTIONS*] [*ARGUMENT*]
 
 # DESCRIPTION
 
-**tpm2_sign**(1) - Signs an externally provided message or hash with the
-specified symmetric or asymmetric signing key.
+**tpm2_sign**(1) - Generates signature of specified message or message-digest
+using the specified symmetric or asymmetric signing key.
 
-If the signing key is a restricted signing key, then validation can be provided
-via the **-t** output. The ticket indicates that the TPM performed the hash of
-the message.
+When signing a message, **tpm2_sign** utility first calculates the digest of the
+message like in the **tpm2_hash**. It also generates a validation ticket under
```

Comment:
Grammer:
```When signing a message, **tpm2_sign** utility first calculates the digest of the
message like in the **tpm2_hash**. It also generates a validation ticket under..``

"like in the tpm2_hash" does not compute. Try:

```When signing a message, **tpm2_sign** utility first calculates the digest of the
message simialr to the **tpm2_hash(1)** command. It also generates a validation ticket under...```